### ISSUE TITLE
Homebrew release

### ISSUE THREAD

**Issue by sluongng** [facebook/dotslash#11]
It would be nice to be able to manage dotslash using homebrew on MacOS

Most other Meta's devtools are released on homebrew.

**Comment by bolinfest** [comment]
Yes, I would certainly like to set this up. Hopefully also apt and dnf for Linux distros.

**Comment by abhinav** [comment]
FYI: https://github.com/Homebrew/homebrew-core/pull/162300

Homebrew maintainers don't usually accept formulas for repositories newer than 30 days, but I've prepped the formula PR anyway.

**Comment by abhinav** [comment]
https://github.com/Homebrew/homebrew-core/pull/162300 landed.  dotslash is now available on Homebrew.
@bolinfest this issue can be closed now.
